src/components/PokemonCard/PokemonCard.jsx

```diff
@@ -0,0 +1,52 @@
+import './PokemonCard.css'
+import {useEffect, useState} from "react";
+import axios from "axios";
+import './../../index.css'
+function PokemonCard({name}) {
+    const [pokemon, setPokemon] = useState({});
+
+    useEffect(() => {
+        async function fetchPokemonCard() {
+            try {
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}/`);
+
+                setPokemon(result.data);
+                console.log(result)
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        fetchPokemonCard();
+    }, []);
+    return (
+        <>
+
+            {Object.keys(pokemon).length > 0 && (
```

Comment:
Vervolgens kan je check hier dan een stuk simpeler
```suggestion
            {pokemon && (
```